### DISCUSSION TITLE
Readthedocs now uses Google Analytics

### DISCUSSION THREAD

**Discussion by comradekingu** [WeblateOrg/weblate#4830] [General] [answered]
Can something be done about this?

**Comment by nijel** [comment]
Really? I don't see it used anywhere on few projects I've checked...

**Comment by comradekingu** [comment]
https://docs.weblate.org/en/latest/
ethicalads.io too, whatever that is

**Reply by nijel** [reply]
Okay, looking at [their docs](https://docs.readthedocs.io/en/stable/advertising/advertising-details.html) explains it:

> Users can opt-out of analytics by using the Do Not Track feature of their browser.

That's what I do in my browser and that's why I don't get it served.

**Comment by nijel** [comment, marked as answer]
Anyway there is one thing I wanted to do for some time and now I finally did - I've just purchased [Gold membership](https://readthedocs.org/sustainability/#gold-membership) there and configured Weblate docs to be ad-free. While doing that I've also disabled Google Analytics for it.
